### DISCUSSION TITLE
Update mentions if values change

### DISCUSSION THREAD

**Discussion by lzinga** [afconsult/quill-mention#323] [Ideas]
So from what I can tell and from my usage it looks like if the html is saved and reloaded it works properly however if the list of values changed it doesn't update the mentions.

For example with the below values
```js
const atValues = [
    { id: 1, value: "Jimmie Dean" },
    { id: 2, value: "Patrik Sjölin" }
];
```
So if the quill html gets saved and reloaded it loads properly however if value of id 1 has changed to a different name it would be nice if it could update all mentions within the quill with that new name since it has the same id this should be possible.

I will probably look into this myself and see how easy it is but hoping other people are interested in this as well.

**Comment by tabrez500** [comment]
Hi @lzinga is there any update on this?

**Comment by solomon23** [comment]
interested in this as well !

**Comment by lzinga** [comment]
No update for it, from what I recall I never found   a way around it. Sorry I couldn't do much from my end about this issue. Hopefully someone more familiar with quill can jump in.
